packages/block-library/src/image/image.js

```diff
@@ -552,6 +572,13 @@ export default function Image( {
 		>
 			{ controls }
 			{ img }
+			{ duotone && (
+				<DuotoneFilter
+					selector={ `.wp-block-image.${ duotone.id } img` }
+					id={ duotone.id }
+					values={ duotone.values }
+				/>
+			) }
 			{ ( ! RichText.isEmpty( caption ) || isSelected ) && (
 				<RichText
 					ref={ captionRef }
```

Comment:
Is there a way to absorb the changes made  here in the support flag, meaning if we want to add support to other blocks, there's no changes that should be made in the block itself?

Reply:
The image block was added separately so the positioning of the toolbar could be controlled. @jasmussen and @mtias said they wanted the duotone toolbar positioned alongside the crop controls or as part of the crop controls which couldn't be done with block supports (See https://github.com/WordPress/gutenberg/pull/26361#issuecomment-714415453 and the following few comments)

Reply:
>  as part of the crop controls which couldn't be done with block supports

This is not possible now sure but after #29247 it will be possible to use the "block" segment for that. I'm fine if it's added to the end personally until that other PR lands.